components/collection/drop/DropContainer.vue

```diff
@@ -7,7 +7,7 @@
       @model-value="isLoading = false" />
     <CountdownTimer />
     <hr class="text-color my-0" />
-    <div class="container is-fluid">
+    <div class="w-full px-[2.5rem]">
```

Comment:
seems `.container.is-fluid` not equal to `.w-full.px-[2.5rem]`:

```css
.container.is-fluid {
    padding-left: 2.5rem;
    padding-right: 2.5rem
}

@media screen and (max-width: 768px) {
    .column-padding {
        margin:0 -12px!important;
        padding-left: 10px!important;
        padding-right: 10px!important
    }

    .container.is-fluid {
        padding-left: 1.25rem;
        padding-right: 1.25rem
    }
}
```

Reply:
@newraina I've double check https://bulma.io/documentation/layout/container/#fluid-container and it look like it's always `32px`

so maybe instead we should have `w-full px-2`

Reply:
Nice find, I totally forgot about:
```scss
$fluid-container-padding: 2.5rem;
$fluid-container-padding-mobile: 1.25rem;
```
in that case I would replace `w-full px-[2.5rem]` to `w-full px-5 md:px-10`